docs/latest/deployment/index.md

```diff
@@ -127,10 +127,85 @@ appropriate headers. You can find an example of this setup in our
 
 #### What if my archive doesn't support DicomWeb?
 
-> This is possible to do with the OHIF Viewer, but not as straightforward. Look
-> out for documentation on this subject in the near future.
+It's possible to supply all Study data via JSON format, in the event you do not have a DicomWeb endpoint.
+You can host all of the relevant files on any web accessible server (Amazon S3, Azure Blob Storage, Local file server etc.)
+
+This JSON is supplied via the '?url=' query parameter.
+It should reference an endpoint that returns **application/json** formatted text.
+
+If you do not have an API, you can simply return a text file containing the JSON from any web server.
+
+
+You tell the OHIF viewer to use JSON by appending the ```'?url='``` query to the ```/Viewer``` route:
```

Comment:
For inline comments, you can use single backticks (\`) around the code. For multi-line code blocks, you can use triple backticks (\`) and specify the language/syntax after the triple backticks For example:

\```js
function doStuff() {
   // stuff to do
}
\```

Reply:
Fixed